### PR TITLE
Add client options in usage

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
Hello, @yoyama, @muga 

What do you think to add this message?
https://github.com/treasure-data/digdag/blob/master/digdag-cli/src/main/java/io/digdag/cli/client/ClientCommand.java#L46-L57

```
  Client options:
    -e, --endpoint URL               Server endpoint (default: http://127.0.0.1:65432)
    -H, --header  KEY=VALUE          Additional headers
    --disable-version-check          Disable server version check
    --disable-cert-validation        Disable certificate verification
```

### CODE REVIEW

File: digdag-cli/src/main/java/io/digdag/cli/Main.java [hunk 1]
```diff
@@ -353,4 +355,14 @@ public static void showCommonOptions(Map<String, String> env, PrintStream err)
         err.println("    --version                        show client version");
         err.println("");
     }
+
+    public static void showClientCommonOptions(Map<String, String> env, PrintStream err)
```
Comment (yoyama) [inline digdag-cli/src/main/java/io/digdag/cli/Main.java]: It looks like the arg. env is not used.

**Comment by yoyama** [comment]
@hiroyuki-sato  Thanks for your proposal.
I think it is basically good.
But I added a comment. Please check it.

File: digdag-cli/src/main/java/io/digdag/cli/Main.java (hunk 1 above)
Comment (hiroyuki-sato) [reply digdag-cli/src/main/java/io/digdag/cli/Main.java]: Removed `env` argument

**Comment by hiroyuki-sato** [comment]
@yoyama Thank you for the comment. Fixed your comment. Please take a look when you have time.

**Pull request merged**